app/scripts-browserify/map.js

```diff
@@ -147,11 +148,9 @@ var Map = {
       let googleMapReference = this.googleMap;
       markers.forEach(marker => {
         marker.addListener('mousedown', function() {
-          var currentMarker = this;
-          $.each(markers, function(i, marker) {
-            if(marker !== currentMarker) {
-              marker.infobox.close();
-            }
+          google.maps.event.trigger(googleMapReference, 'closeAllInfoboxes');
```

Comment:
Throw in a single comment - I'm guessing most people would do a double-take when they saw a trigger and addListener for the same event side-by-side